.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-  - package-ecosystem: "pip-compile"
```

Comment:
Weird, I just saw an example of using this [here](https://github.com/mozilla-releng/tooltool/blob/fb7d180f5c21286f443a709e57538643fd8b6707/.github/dependabot.yml).